vortex-buffer/src/lib.rs

```diff
@@ -1,3 +1,15 @@
+#![deny(missing_docs)]
+
+//! A byte buffer implementation for Vortex.
+//!
+//! Vortex arrays hold data in a set of buffers.
+//!
+//! # Alignment
+//!
+//! We do not currently enforce any alignment guarantees on the buffer. This can be problematic
```

Comment:
Maybe worth linking https://github.com/spiraldb/vortex/issues/115 re: alignment in case this comment goes stale. Also, the recommendation to convert to Arrow doesn't really help, since it accepts arbitrarily aligned vecs? (only enforces alignment for things it allocates)